docs/sw-update-conf.json

```diff
@@ -15,28 +15,49 @@
     "restart": "environment"
   },
   "octoprint": {
-    "pip": "https://github.com/mrbeam/OctoPrint/archive/{target_version}.zip",
-    "prerelease": false,
+    "type": "github_release",
     "develop":{
+      "type": "github_commit"
+    },
+    "beta": {
+      "prerelease_channel": "mrbeam2-{tier}",
       "prerelease": true
     },
-    "beamos_date": {
-      "2021-06-11": {
-        "prerelease": true
-      }
```

Comment:
What was this beamos_date used for and how? Why don't we need it anymore?

Reply:
this was used by the buster image, this used at the beginning a separate branch